staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -989,6 +989,24 @@ type KubeletConfiguration struct {
 	// +featureGate=GracefulNodeShutdown
 	// +optional
 	ShutdownGracePeriodCriticalPods metav1.Duration `json:"shutdownGracePeriodCriticalPods,omitempty"`
+	// ShutdownGracePeriodByPodPriority specifies the shutdown grace period for Pods based on their associated priority class value. Requires the GracefulNodeShutdown feature gate to be enabled.
```

Comment:
Remove one of the two repeated "Requires..." and move that line directly below this one.